app.rb

```diff
@@ -1,20 +1,143 @@
+require_relative "models/apartment"
+require_relative "models/tenant"
 require_relative "data"
+
 apartments = data[:apartments]
 tenants = data[:tenants]
 
-# First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
-  # Explain how the data is structured
-  # What are the properties for each of the two types of hashes
+ruby_apartments = []
+ruby_tenants = []
+
+apartments.each do |apartment|
+new_apts = Apartment.new(apartment[:id], apartment[:address], apartment[:monthly_rent], apartment[:square_feet])
```

Comment:
only suggestion would be make this variable singular `new_apt` for clarity. That's because while it changes over time (as the `each` iterates over the array), it only ever holds one apartment object at a time.